man/systemd.netdev.xml

```diff
@@ -456,6 +456,16 @@
           <xi:include href="version-info.xml" xpointer="v257"/>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>NoLinkLocalLearning=</varname></term>
```

Comment:
Please make the setting positive, i.e. `LinkLocalLearning=`.
We typically do not use negative words in setting.